app/views/relatives/_show.html.slim

```diff
@@ -5,6 +5,7 @@ tr
       dl.dl-horizontal
         - relative.relatives.each do |rel|
           dt= rel.full_name
-          dd
-            = l(rel.date_of_birth) if rel.date_of_birth
-            |, #{rel.relation}
+          -unless rel.full_name.blank?
```

Comment:
How can `rel.full_name` ever be `blank?` after we have `validates :first_name, :last_name, :email, presence: true` in the model?